### PR TITLE
Fix preview

### PR DESCRIPTION
**Pull request by CamLamb**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -14,7 +14,7 @@ django-crispy-forms = "^1.14.0"
 django-environ = "^0.10.0"
 # django_redis = "^5.2.0"
 django-settings-export = "^1.2.1"
-django-simple-history = "^3.3.0"
+django-simple-history = { git = "https://github.com/jazzband/django-simple-history.git", rev = "29108a5" }
```
Comment (CamLamb) [inline pyproject.toml:17]: django-simple-history needs a new release to fix this issue, in the meantime we can pin a commit.

For added protection, we should fork the repo and point this at the head on our own fork

Comment (marcelkornblum) [reply pyproject.toml:17]: agree with the forking approach. annoying moving to a git ref :(

what's the actual issue?

Comment (CamLamb) [reply pyproject.toml:17]: https://github.com/jazzband/django-simple-history/pull/1256
TLDR: they removed a condition that was thought to be unnecessary, but was actually needed in some scenarios (Wagtail Preview managed to hit this scenario)

**Review by marcelkornblum — APPROVED** [review]
